addons/mail/static/src/models/thread/thread.js

```diff
@@ -1630,6 +1630,24 @@ function factory(dependencies) {
             }
         }
 
+        /**
+         * Message from mailing channel should not make a notification in
+         * Odoo for users with notification "Handled by Email".
```

Comment:
I feel like this explains the condition rather than the function, so I would suggest moving it above the condition.